velox/vector/tests/VectorToStringTest.cpp

```diff
@@ -302,4 +302,9 @@ TEST_F(VectorToStringTest, printIndices) {
   EXPECT_EQ(
       printIndices(indices), "5 unique indices out of 6: 34, 79, 11, 0, 0, 33");
 }
+
+TEST_F(VectorToStringTest, indexOverflow) {
+  auto flat = makeFlatVector<int32_t>({1, 2, 3});
+  ASSERT_THROW(flat->toString(4), VeloxException);
```

Comment:
Hi Jin, Thank you for improving the error messages. Could we check that the new error message does contain the index and valid range as expected in this test?